var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -70,9 +70,14 @@ def __build_mods_coreneuron(self, mods_location, link_flag, include_flag):
         spec = self.spec
         assert os.path.isdir(mods_location), mods_location
         include_flag += ' -I%s' % (spec['coreneuron'].prefix.include)
-        which('nrnivmodl-core')(
-            '-i', include_flag, '-l', link_flag, '-n', self.mech_name,
-            '-v', str(spec.version), '-c', mods_location)
+        if spec.satisfies('coeneuron@develop') or spec.satisfies('coeneuron@master'):
```

Comment:
I was using this branch and I came across a small issue. I think this should be:
```suggestion
        if spec.satisfies('^coreneuron@develop') or spec.satisfies('^coreneuron@master'):
```

Reply:
You're right!